flipperdevices/flipperzero-firmware
# [FL-1885] USB-UART bridge

# What's new

- USB-UART bridge application
- LPUART driver

# Verification 

- Go to *GPIO->USB-UART bridge*
- Set port configuration and select *enable* to start bridge
- Short Rx/Tx or C0/C1 (depending on selected port)
- Connect to VCP port 0/1 (depending on selected channel), data sent to console should return back
- Stop bridge by selecting *disable* or by pressing *back*

# Checklist (do not modify)
